yasa-ui/src/style/iconfont/iconfont.css

```diff
@@ -0,0 +1,121 @@
+  url('iconfont.woff?t=1594114486700') format('woff'),
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/yasa-org/yasa/pullRequest?prid=5815330)